src/ui/components/CardDetails/CardDetailsItem/CardDetailsItem.tsx

```diff
@@ -75,12 +71,29 @@ const CardDetailsItem = ({
           )}
           {info}
         </IonText>
+        {actionIcon && (
+          <IonButton
+            slot="end"
+            shape="round"
+            className="action-button"
+            data-testid={`${testId}-action-icon`}
+            onClick={actionIconClick}
+          >
+            <IonIcon icon={actionIcon} />
+          </IonButton>
+        )}
         {copyButton && (
           <IonButton
             slot="end"
             shape="round"
-            className="copy-button"
+            className="action-button"
             data-testid={`${testId}-copy-button`}
+            onClick={() => {
+              if (copyButton) {
+                writeToClipboard(info);
+                dispatch(setToastMsg(ToastMsgType.COPIED_TO_CLIPBOARD));
+              }
+            }}
```

Comment:
Could you please explain the two things here? Why the `copyButton` has class name `action-button` instead of the old `copy-button`? can each of them have their own class name?

Also, line 92 `if (copyButton)` that is always true because we have `copyButton &&` at line 85, right?

Can we just pass a function name here for the `onClick` event and then move everything inside the new function somewhere in the page above?

Reply:
Because 2 buttons have same styles, I think they should have same class name and this name meaningful to both. (`copy` also just action of this component)